broker-node/app/ChunkEvents.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+
+namespace App;
+
```

Comment:
Nit: Don't double space code. Although we don't have a style guide yet for PHP, code should be somewhat consistent.